parsl/executors/base.py

```diff
@@ -87,6 +88,83 @@ def scaling_enabled(self) -> bool:
         """
         pass
 
+    @abstractmethod
+    def status(self) -> Dict[Any, JobStatus]:
+        """Return the status of all jobs/blocks currently known to this executor.
+
+        :return: a dictionary mapping job ids to status strings
+        """
+        pass
+
+    @property
+    @abstractmethod
+    def status_polling_interval(self) -> int:
+        """Returns the interval, in seconds, at which the status method should be called. The
+        assumption here is that, once initialized, an executor's polling interval is fixed.
+        In practice, at least given the current situation, the executor uses a single task provider
+        and this method is a delegate to the corresponding method in the provider.
+
+        :return: the number of seconds to wait between calls to status() or zero if no polling
+        should be done
+        """
+        pass
+
+    @property
+    @abstractmethod
+    def error_management_enabled(self) -> bool:
+        """Indicates whether worker error management is supported by this executor. Worker error
+        management is done externally to the executor. However, the executor must implement
+        certain methods that allow this to function. The basic idea of worker error management
```

Comment:
what are the certain methods? Is it the _get_job_ids abstract method on StatusHandlingMixin or more?

Reply:
I'll add a note.